apps/volunteer/main.py

```diff
@@ -21,3 +26,78 @@ def safeguarding():
 @feature_flag("VOLUNTEERS_SIGNUP")
 def info():
     return render_template("volunteer/info.html")
+
+
+@volunteer.route("/init_shifts")
```

Comment:
[can be merged] This should probably be in a tasks file (e.g. https://github.com/emfcamp/Website/blob/main/apps/cfp/tasks.py)

Reply:
Went with view rather than CLI task to trigger this as I can hit the URL for it but don't have access to run CLI tasks.